### DISCUSSION TITLE
Hx711

### DISCUSSION THREAD

**Discussion by iyssoft** [gpiozero/gpiozero#1055] [General]
I have 2 Raspberry Pi4, running as first one is master and the second one is slave. They are connected via ethernet cable. A loadcell and HX711 have been wired to the slave I would like to read the weight data from the master. GPIO Zero library(https://gpiozero.readthedocs.io/en/stable/recipes_remote_gpio.html) Has few examples Led, Button etc. as master and slave I could not find an example for Hx711. Does any one has experience with GPIO Zero library and Hx711 server, client solution.

Thanks

**Comment by bennuttall** [comment]
There's no built-in support for this particular device, but you can devise the implementation yourself.

There's an example here using RPi.GPIO: https://github.com/tatobari/hx711py/blob/master/hx711.py

You can take this code, replace the RPi.GPIO calls to use the gpiozero equivalents (`InputDevice` and `OutputDevice`) and it should "just work", even with remote pins.

**Reply by iyssoft** [reply]
I tried to make changest, but I always read 0 value.
